windows/src/global/delphi/general/Upload_Settings.pas

```diff
@@ -32,12 +32,11 @@ interface
 const
   // https://api.keyman.com/ - programmatic endpoints
   API_Path_UpdateCheck_Windows = '/windows/14.0/update'; // version will only update when the api changes
-
-  // TODO: use /windows/ instead of /desktop/
-  API_Path_UpdateCheck_Desktop = '/desktop/'+SKeymanVersion+'/update';  // TODO: use updatecheck_windows
```

Comment:
We're deleting `API_Path_UpdateCheck_Desktop`?  Is that intended?

Reply:
Yes, that's correct -- we're moving towards `/windows` as the API endpoint root for Windows, rather than `/desktop`.